photo/urls.py

```diff
@@ -17,6 +18,7 @@
     path('/related-collection/<photo_id>', RelatedCollectionView.as_view()),
     path('',PhotoView.as_view()),
     path('/back/<collection_name>',BackgroundView.as_view()),
+    path('/collection-main', CollectionMainView.as_view()),
```

Comment:
main 은 엔드포인트 URL에  포함안해도 될것 같지만, main 의 의미를 꼭 주고싶으면 main-collection 추천 드립니다.

```suggestion
    path('/main-collection, CollectionMainView.as_view()),
```

Reply:
수정 하였습니다